projects/pastanaga/src/lib/svg/icon.component.ts

```diff
@@ -46,8 +69,14 @@ export class IconComponent implements OnChanges {
 
     private setSvg(svg: SVGElement) {
         const icon = <SVGElement>svg.cloneNode(true);
-        if (typeof this.hidden !== 'undefined') {
-            this.renderer.setAttribute(icon, 'aria-hidden', this.hidden.toString());
+        if (typeof this._hidden !== 'undefined') {
```

Comment:
why not just `this._hidden !== 'undefined'`? :thinking:

Reply:
because I used to work in a place where `undefined` as a keyword was forbidden by our linter 😉